src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
From the 'Common mistakes" Do not forget to add https://github.com/target to custom annotations.